releases/0.4.0/index.md

```diff
@@ -4,16 +4,58 @@ layout: releases
 custom_css: releases
 ---
 
-## {{ page.title }}
+# {{ page.title }}
 
-###### Date: _TBD_
+##### Date: 2021-05-27
 
-#### Ontology File(s)
+## Ontology File(s)
 
-[GitHub](https://github.com/casework/CASE/tree/develop)
+[GitHub](https://github.com/casework/CASE/releases/tag/0.4.0)
 
-#### Migration guidance
+## Migration guidance
 
 A guide to migration from CASE 0.3.0 will be provided.
 
-#### Release Notes
+## Release Notes
+
+### Focus
```

Comment:
There are two foci.  You noted one.  The other is upgrading to UCO 0.6.0, which includes changes brought forward by the CASE community.

This could be used as a link to UCO 0.6.0:
https://unifiedcyberontology.org/releases/0.6.0/